src/main/java/graphql/execution/instrumentation/dataloader/PerLevelDataLoaderDispatchStrategy.java

```diff
@@ -125,6 +119,10 @@ public void executionStrategyOnFieldValuesException(Throwable t, ExecutionStrate
 
     @Override
     public void executeObject(ExecutionContext executionContext, ExecutionStrategyParameters parameters) {
+        if (executionContext.getIncrementalCallState().getIncrementalCallsDetected()) {
```

Comment:
why not in `executionStrategy` which happens before?

Reply:
Hm `executionStrategy` executes before any field is executed, and we need some fields to be executed in order to identify if `@defer` is being used.